opsdroid/connector/tests/test_connector_rocketchat.py

```diff
@@ -0,0 +1,295 @@
+"""Tests for the RocketChat connector."""
+import asyncio
+import pytest
+import contextlib
+import asynctest.mock as amock
+
+from opsdroid.core import OpsDroid
+from opsdroid.connector.rocketchat import RocketChat
+from opsdroid.events import Message
+from opsdroid.cli.start import configure_lang
+
+configure_lang({})
+
+
+def test_init():
+    """Test that the connector is initialised properly."""
+    connector = RocketChat(
+        {
+            "name": "rocket.chat",
+            "token": "test",
+            "user-id": "userID",
+            "update-interval": 0.1,
+        },
+        opsdroid=OpsDroid(),
```

Comment:
We should use the `opsdroid` fixture as an argument and pass it here. The same with all the other cases